packages/gasket-plugin-metadata/lib/actions.js

```diff
@@ -62,7 +63,20 @@ async function getMetadata(gasket) {
         pluginData.metadata.path = path.join(app.metadata.path, 'plugins');
         plugins.push(pluginData);
       } else {
-        pluginData.metadata.path = path.dirname(path.join(require.resolve(pluginData.name), '..'));
+        let resolvedPath;
+        try {
+          resolvedPath = require.resolve(pluginData.name, { paths: [gasket.config.root] });
+        } catch (err) {
+          // try esm import
+          try {
+            resolvedPath = await import(pluginData.name);
+          // eslint-disable-next-line no-catch-shadow, no-shadow
+          } catch (err) {
+            gasket.logger.error(`Error resolving plugin ${pluginData.name}: ${err.message}`);
```

Comment:
IMO errors resolving plugins for getting metadata should not crash the app but can revert if needed